app/components/publishers/vacancies_component.rb

```diff
@@ -46,13 +46,15 @@ def view_applicants(vacancy, card)
     return unless vacancy.enable_job_applications?
     return unless include_job_applications?
 
-    if vacancy.job_applications.any?
-      link = govuk_link_to(I18n.t("jobs.manage.view_applicants", count: vacancy.job_applications.count),
+    if post_submission_job_applications(vacancy.job_applications).present?
+      link = govuk_link_to(I18n.t("jobs.manage.view_applicants", count: post_submission_job_applications(vacancy.job_applications).count),
```

Comment:
Minor performance improvement:

If you use an instance variable, you can avoid having to run the `post_submission_job_applications` database query twice.

That instance variable could be set in a controller helper_method or in the controller action itself.